### ISSUE TITLE
Leon Sans

### ISSUE THREAD

**Issue by cmiscm** [google/fonts#2169]
Hi, I've gotten a message from @davelab6  to contribute my Leon Sans to GF.
You can download all the fonts and texts : [leonsans.zip](https://github.com/google/fonts/files/3618524/leonsans.zip)

I checked CONTRIBUTING.md and added all the glyphs it requires, and I runned Font Bakery for the font. There's some failed message, but it looks like not critical issues for the font. See the json file -  [report.json.zip](https://github.com/google/fonts/files/3618527/report.json.zip)

Please let me know if there's anything I need to do, also you can contact me cmiscm@google.com.
Thanks.

**Comment by m4rc1e** [comment]
Thanks. Can you create a new github repository for this project please? it should be structured in a similar fashion to https://github.com/googlefonts/OswaldFont.

**Comment by m4rc1e** [comment]
I'm guessing https://github.com/cmiscm/leonsans is the repo?

We should tag a release in the repo which contains ttfs and otfs.

**Comment by cmiscm** [comment]
Here's github repository.
https://github.com/cmiscm/leonsansfont
The repo you mentioned is for js code, so I created new one for the structure you required.

**Comment by cmiscm** [comment]
Hi, is there any update for this? Can I know the progress? If there's anything I need to do, please let me know!

**Comment by m4rc1e** [comment]
Have you checked the fonts using [fontbakery](https://github.com/googlefonts/fontbakery)? the family should produce no failing checks.

**Comment by cmiscm** [comment]
Yes, I checked Font Bakery for the font. There's some failed message, but it looks like not critical issues for the font. See the json result file - https://github.com/google/fonts/files/3618527/report.json.zip

**Comment by cmiscm** [comment]
It's been a year, and any updates for this?

**Comment by telmen** [comment]
Ye, is there any progress going on this?

**Comment by stochastic-thread** [comment]
Any updates on this?
